tests/fixtures/addon/npm/.eslintrc.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
       excludedFiles: [
         'app/**',
         'addon/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
+        'addon-test-support/**'
```

Comment:
@rwjblue there is a change for `addon` fixtures for yarn and npm blueprints. PR passes current tests suite which means there are no changes to `app` blueprint. Travis CI failed because of eslint timed out on `lib` folder check(it has been fixed https://github.com/ember-cli/ember-cli/commit/18b59f80d0e42380ab3c618c472f87f7e0baa116).
Let me know please if I need to fix something. Thanks.